webpack/components/AnsibleHostDetail/components/AnsibleHostInventory/AnsibleHostInventory.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import AnsibleHostInventory from "./AnsibleHostInventory";
+
+
+describe('AnsibleHostInventory', () => {
+    it('should show content', () => {
+        render(<AnsibleHostInventory />);
```

Comment:
`render` only renders the component but does not actually do any asserts. A basic check that the expected text is shown should be enough (following example not tested):

```js
 import { screen } from '@testing-library/react'
import '@testing-library/jest-dom'; // might be needed for toBeInTheDocument method

expect(screen.getByText('_meta')).toBeInTheDocument();
```